### PR TITLE
feat added transferOwnership route

### PR DESCRIPTION
**Pull request by Igor-aka-Gillirond**

### CODE REVIEW

File: src/dev-team.js [hunk 1]
```diff
@@ -23,5 +23,9 @@ export default req => ({
 
   updateNotificationSettings(appId, settings) {
     return req.put(`${urls.appConsole(appId)}/notifications/limits`, settings)
+  },
+
+  transferOwnership(appId, devId) {
+    return req.put(`${devTeam(appId, 'owner')}`, devId)
```
Comment (Valodya) [inline src/dev-team.js]: `devId` looks like just a String, but `PUT` method is expecting an`Object`

File: src/dev-team.js [hunk 2]
```diff
@@ -23,5 +23,11 @@ export default req => ({
 
   updateNotificationSettings(appId, settings) {
     return req.put(`${urls.appConsole(appId)}/notifications/limits`, settings)
+  },
+
+  transferOwnership(appId, devId) {
```
Comment (Valodya) [inline src/dev-team.js]:
```
transferOwnership(appId, developerId) {
    return req.put(`${devTeam(appId, 'owner')}`, { developerId })
}
```

File: src/dev-team.js (hunk 1 above)
Comment (Igor-aka-Gillirond) [reply src/dev-team.js]: Done

File: src/dev-team.js (hunk 2 above)
Comment (Igor-aka-Gillirond) [reply src/dev-team.js]: Done

**Pull request merged**